src/More/About.tsx

```diff
@@ -18,9 +12,11 @@ export const AboutScreen: FunctionComponent = () => {
   const { t } = useTranslation()
 
   const osInfo = `${Platform.OS} v${Platform.Version}`
-  const pathCheckWebAddress = "pathcheck.org"
-  const pathCheckUrl = "https://pathcheck.org/"
   const { applicationName, versionInfo } = useApplicationInfo()
+  const healthAuthorityName =
```

Comment:
@aledustet Will all this stuff be wrapped up in the config context? It's starting to feel pretty brittle.

Reply:
Definitely, this all os a candidate, for this screen however there might be changes coming soon but we should have this done soon.